nshlib/nsh_builtin.c

```diff
@@ -234,7 +234,13 @@ int nsh_builtin(FAR struct nsh_vtbl_s *vtbl, FAR const char *cmd,
 
           /* Restore the old actions */
 
-          sigaction(SIGCHLD, &old, NULL);
+#  ifndef CONFIG_SCHED_WAITPID
```

Comment:
don't need "# ifndef CONFIG_SCHED_WAITPID", keep align with line 92

Reply:
line 135 already have this check if CONFIG_SCHED_WAITPID is defined